src/index.html

```diff
@@ -8,6 +8,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <ul class="stars stars--0">
+      <li class="stars__star"></li>
```

Comment:
```suggestion
     <div class="stars stars--0">
      <div class="stars__star"></div>
```
it is not list
in this case better to make div.
also will be no need to reset list styles